### DISCUSSION TITLE
Problème appel API CreerFormulaireIndividu - 404

### DISCUSSION THREAD

**Discussion by warmup79** [MTESSDev/FRW#278] [Annonces]
Bonjour,

Nous sommes à créer notre premier formulaire et nous avons systématiquement une erreur 404 à chaque fois qu'on appelle l'API CreerFormulaireIndividu à l'URI : https://formulaires.it.mtess.gouv.qc.ca/fr/api/v1/SIS/CreerFormulaireIndividu/Q05_calendrier

Le formulaire Q05_calendrier existe bien et nous sommes en mesure de l'ouvrir depuis la page des essais.

![image](https://github.com/user-attachments/assets/842a19c7-c808-40d3-98e7-ad511edbc2b2)

En vérifiant la documentation dans github tout semble en règle. Êtes-vous en mesure de nous aider à comprendre ce qu'on fait de pas correct ?
![image](https://github.com/user-attachments/assets/f2c801cb-e2ca-4e71-a5d6-48d5e0d05fd7)

**Comment by loima02** [comment]
Bonjour,

Avez-vous validé vos paramètres à l'aide de notre Swagger (voir ici : https://formulaires.it.mtess.gouv.qc.ca/swagger/index.html)?

Pouvez-vous me donner la date et l'heure de votre dernier essai?

Merci et bonne journée.

**Comment by warmup79** [comment]
Bonjour,

Merci avec le Swagger, j'ai pu déterminer qu'il n'y a pas de /fr/ dans l'URL :

***@***.***

Merci


Patrick Morneau
Téléphone : (418) 646-0425 poste 80924
***@***.******@***.***>

De : Mathieu Loiselle ***@***.***>
Envoyé : 22 juillet 2024 13:25
À : MTESSDev/FRW ***@***.***>
Cc : Morneau, Patrick ***@***.***>; Author ***@***.***>
Objet : [EXTERNE] Re: [MTESSDev/FRW] Problème appel API CreerFormulaireIndividu - 404 (Discussion #278)

*ATTENTION : Ce courriel provient de l'extérieur de votre organisation.
Évitez de cliquer sur un hyperlien, d'ouvrir une pièce jointe ou de transmettre des informations personnelles si vous ne connaissez pas l'expéditeur du courriel. En cas de doute, communiquez verbalement avec lui.

Bonjour,

Avez-vous validé vos paramètres à l'aide de notre Swagger (voir ici : https://formulaires.it.mtess.gouv.qc.ca/swagger/index.html)?

Pouvez-vous me donner la date et l'heure de votre dernier essai?

Merci et bonne journée.

-
Reply to this email directly, view it on GitHub<https://github.com/MTESSDev/FRW/discussions/278#discussioncomment-10117439>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BI2SZJTR6KACA7CBHFWREJDZNU56XAVCNFSM6AAAAABLITL5EWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMJRG42DGOI>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>

________________________________
Ce message est confidentiel et est à l'usage exclusif du destinataire nommé ci-dessus. Si vous recevez ce message par erreur, sachez qu'il est interdit de le diffuser, de le distribuer, d'en dévoiler le contenu ou de le reproduire. Veuillez informer immédiatement l'expéditeur par courrier électronique et détruire l'original de ce message ainsi que toute copie.
________________________________
